gui/src/main/java/de/azubi/kniffel/gui/fx/table/ScoreRow.java

```diff
@@ -0,0 +1,82 @@
+package de.azubi.kniffel.gui.fx.table;
+
+import java.lang.reflect.Field;
+import java.util.Objects;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class ScoreRow {
+    private Object scoreName;
+    private Object player1;
```

Comment:
Warum ist das kein Array?
Warum so ein allgemeiner Typ Object?
Habt ihr keinen Player?

Reply:
Bei uns gibt es allgemein Player 1 ... und dann nochmal 2 -5 

So können wir garantieren, dass es am Ende auch Multiplayer und Einzelplayer also Auswahlmöglichkeit gibt...

Reply:
Weil wenn man bei einem Spieler ja nur ist, muss das Feld von der Darstellung her auch eine andere sein und da haben wir es dann so gemacht, geht unserer Meinung nach n bissl simpler so... Hoffen das passt?

Reply:
Wir haben uns dazu beschlossen ein issue dafür aufzumachen und das mal hintenanzustellen ... das werden wir heute nachmittag in angriff nehmen ... bis dahin würden wir mit objects weiterarbeiten... aber am ende , haben wir unsere eigene superklasse!